src/Pages/Homepage/index.js

```diff
@@ -19,12 +19,28 @@ const Homepage = () => {
   return (
     <div class='homepage-category'>
       <div class='hpg-genre-container-1'>
-        <HomepageCard className='hpg-genre-1' to='#' title='Fiction' />
-        <HomepageCard className='hpg-genre-2' to='#' title='Tutorials' />
-        <HomepageCard className='hpg-genre-3' to='#' title='Non Fiction' />
+        <HomepageCard
+          className='hpg-genre-1'
```

Comment:
suggestion: keep the components separately from pages like creating pages and components folders.